pom.xml

```diff
@@ -107,9 +108,66 @@
 				<version>${eventVersion}</version>
 				<scope>provided</scope>
 			</dependency>
+
+			<dependency>
+				<groupId>org.springframework.ws</groupId>
```

Comment:
shouldn't these dependencies be added in api pom?

Reply:
No, it's not required